### PR TITLE
Add a simple tool to get the current size of a VHD

### PR DESCRIPTION
**Pull request by jonludlam**
Signed-off-by: Jon Ludlam <jonathan.ludlam@citrix.com>

### CODE REVIEW

**Comment by jonludlam** [comment]
This might break the RPM build until the spec file is updated

File: lib_test/get_vhd_vsize.ml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+let get_vhd_vsize filename =
+  Vhd_lwt.IO.openfile filename false >>= fun fd ->
+  let rec loop = function
+    | End -> return ()
+    | Cons (hd, tl) ->
+      begin match hd with
+      | Fragment.Footer x ->
+	let size = x.Footer.current_size in
+	Printf.printf "%Ld\n" size;
+	exit 1
```
Comment (simonjbeaumont) [inline lib_test/get_vhd_vsize.ml]: Shouldn't this be `exit 0`...?

File: lib_test/get_vhd_vsize.ml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+open Vhd.F
+open Vhd_lwt.IO
+
+module In = From_input(Input)
+open In
+
+let get_vhd_vsize filename =
+  Vhd_lwt.IO.openfile filename false >>= fun fd ->
+  let rec loop = function
+    | End -> return ()
```
Comment (simonjbeaumont) [inline lib_test/get_vhd_vsize.ml]: And this be `exit <non-zero>`?

**Comment by jonludlam** [comment]
Not the right place for this code anyway.

**Comment by simonjbeaumont** [comment]
Sure, but you ported it with the same problems? ;)

**Comment by jonludlam** [comment]
Dammit I knew there was something I needed to do!

**Pull request closed without merging**